__tests__/command.test.ts

```
@@ -0,0 +1,384 @@
+                cmd: Command,
+                readLines: (hymn: string) => Promise<string[]>
+            ): Promise<CommandResult> => {
+                state.lastError = "";
+
+                switch (cmd.cmd) {
+                    case "load":
+                        if (!cmd.hymn) {
+                            state.lastError = "Please enter a hymn number.";
+                            return { success: false, error: state.lastError };
+                        }
+                        const lines = await readLines(cmd.hymn);
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)